services/brig/deb/opt/brig/templates/user/en/email/activation.html

```diff
@@ -48,7 +48,7 @@
 
           <p style="
               color: #000000;">
-            <strong>${email}</strong> was used to create a Wire account. We want to verify that you are indeed <strong>${name}</strong>.
+            <strong>${email}</strong> was used to create a Wire account. Please, verify your email.
```

Comment:
Is it really necessary to emphasize the 'please' part by introducing a comma here? To me, no comma reads more natural in this context (see https://english.stackexchange.com/questions/389/in-what-cases-should-i-use-a-comma-after-please)